pkg/apis/addtoscheme_istio_v1alpha1.go

```diff
@@ -15,10 +15,10 @@
 package apis
 
 import (
-	"istio.io/operator/pkg/apis/istio/v1alpha1"
+	"istio.io/operator/pkg/apis/istio/v1alpha2"
```

Comment:
@ostromart I noticed this go file is still called *_v1alpha1.go, do we need to change it?